<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cp2kInputDialog</class>
 <widget class="QDialog" name="Cp2kInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CP2K Input</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>591</width>
     <height>171</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>The cutoff of the finest grid level</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="basicsTab">
    <attribute name="title">
     <string>Basics</string>
    </attribute>
    <widget class="QLineEdit" name="projectNameLine">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>451</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enter your project name.</string>
     </property>
    </widget>
    <widget class="QLabel" name="projectNameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Project Name:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="runTypeCombo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Energy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Energy and Force</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Molecular Dynamics</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Geometry Optimization</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Monte Carlo</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="runTypeLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>50</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Run Type:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="forceChoiceBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Electronic calculation method</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Force</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QRadioButton" name="mmRadioButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Moleular Mechanics (Classical)</string>
      </property>
      <property name="text">
       <string>MM</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="qmRadioButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>86</width>
        <height>16</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Quantum Mechanics</string>
      </property>
      <property name="text">
       <string>QM</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="mmTab">
    <property name="toolTip">
     <string>Molcular Mechanics (Classical)</string>
    </property>
    <attribute name="title">
     <string>MM</string>
    </attribute>
    <widget class="QComboBox" name="ewaldTypeCombo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>no Ewald</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ewald</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PME</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SPME</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="ewaldTypeLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ewald Type:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="qmTab">
    <property name="toolTip">
     <string>Quantum Mehanics</string>
    </property>
    <attribute name="title">
     <string>QM</string>
    </attribute>
    <widget class="QComboBox" name="qmMethodCombo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>DFT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DFTB (scc)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DFTB (non-scc)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SE (PM6)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="qmMethodLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>12</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Method:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="nMGridSpin">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="nMGridLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>#MultiGrid:</string>
     </property>
    </widget>
    <widget class="QLabel" name="cutOffLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>CutOff (Ry):</string>
     </property>
     <property name="text">
      <string>CutOff(Ry):</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="cutOffSpin">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>The cutoff of the finest grid level</string>
     </property>
    </widget>
    <widget class="QLabel" name="basisLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>42</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Basis:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="basisCombo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="functionalLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Functional:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="functionalCombo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="moreTab">
    <attribute name="title">
     <string>More</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QTextEdit" name="previewText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>591</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="generateButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
